Sources/SPMBuildCore/Plugins/PluginContextSerializer.swift

```diff
@@ -236,11 +236,11 @@ internal struct PluginContextSerializer {
             case .fileSystem(let path):
                 return .local(path: try serialize(path: path))
             case .localSourceControl(let path):
-                return .repository(url: path.asURL.absoluteString, displayVersion: String(describing: package.manifest.version), scmRevision: String(describing: package.manifest.revision))
+                return .repository(url: path.asURL.absoluteString, displayVersion: package.manifest.version?.description ?? "nil", scmRevision: package.manifest.revision ?? "nil")
```

Comment:
happy to fall back to something other than `"nil"` – I was attempting to preserve existing behavior when the values are `nil`, which may or may not be desirable.

Reply:
Since this is "display" version/revision it might be better to say "no version" and "no revision" for `nil` case. 

cc @dschaefer2